pyart/aux_io/gamic_hdf5.py

```diff
@@ -300,8 +319,9 @@ def _get_instrument_params(gfile, filemetadata):
     instrument_params['unambiguous_range'] = dic
 
     dic = filemetadata('nyquist_velocity')
```

Comment:
I would move this inside the if block, there is no reason to create the dictionary if it is not used.